backend/README.md

```diff
@@ -0,0 +1,11 @@
+# Click-n-Split
+### Split expenses with your friends by uploading a picture of shop receipt by using our app. 
+
+## Group Members (Github ID)
+1. Sarvesh Biradar (sarvesh2910)
+2. Pranav Dharamthok (Pranav-PSU)
+3. Supreeth Mudduchetty (Supreeth10)
+4. Vuddagiri Naga Venkata Sai Tejesh (VnvsTejesh) 
+
+## Language
+We are using Javascript for this project as the team members would like to explore Javascript as a server side runtime environment.  
```

Comment:
We should update the readme with how to run the project for someone looking to fork it and provide any other information that might help set up, run and use this project. For example, how many free API hits we have as well as how to create a new account and which files need to be updated with the new API keys.

Reply:
I Agree!^